g_docs.go

```diff
@@ -32,11 +32,10 @@ import (
 	"strings"
 	"unicode"
 
-	"golang.org/x/mod/modfile"
-	"gopkg.in/yaml.v3"
-
 	"github.com/astaxie/beego/swagger"
 	"github.com/astaxie/beego/utils"
+	"golang.org/x/mod/modfile"
+	"gopkg.in/yaml.v3"
```

Comment:
@adirizka7 , is this intentional? :raised_back_of_hand:

Reply:
Yes it is intentional, it was ungrouped before for some reason, but I believe it should be grouped since `"github.com/astaxie/beego/..."` is an external package.

```
import (
    stdlib

    external lib

    internal lib
)
```